packages/block-library/src/tag-cloud/edit.js

```diff
@@ -32,8 +41,26 @@ const MIN_TAGS = 1;
  */
 const MAX_TAGS = 100;
 
+const MIN_FONT_SIZE = 1;
```

Comment:
Let's change this to something smaller to accommodate use cases with `em/rem`. 
```suggestion
const MIN_FONT_SIZE = 0.1;
```